modules/tracking/include/opencv2/tracking/kalman_filters.hpp

```diff
@@ -0,0 +1,215 @@
+    CV_PROP_RW int dataType;                               //!< Type of elements of vectors and matrices, default is CV_64F.
+
+    CV_PROP_RW Mat stateInit;                              //!< Initial state, DP x 1, default is zero.
+    CV_PROP_RW Mat errorCovInit;                           //!< State estimate cross-covariance matrix, DP x DP, default is identity.
+
+    CV_PROP_RW Mat processNoiseCov;                        //!< Process noise cross-covariance matrix, DP x DP.
+    CV_PROP_RW Mat measurementNoiseCov;                    //!< Measurement noise cross-covariance matrix, MP x MP.
+
+    // Parameters of algorithm
+    CV_PROP_RW double alpha;                               //!< Default is 1e-3.
+    CV_PROP_RW double k;                                   //!< Default is 0.
+    CV_PROP_RW double beta;                                //!< Default is 2.0.
+
+    //Functions
+    CV_PROP_RW UKFStateFunction f;                            //!< Function for computing the next state from the previous state.
+    CV_PROP_RW UKFMeasFunction h;                             //!< Function for computing the measurement from the state.
+
+    /** The constructors.
+    */
+    CV_WRAP UnscentedKalmanFilterParams(){}
+
+    /**
+    * @param dp - dimensionality of the state vector,
+    * @param mp - dimensionality of the measurement vector,
+    * @param cp - dimensionality of the control vector,
+    * @param processNoiseCovDiag - value of elements on main diagonal process noise cross-covariance matrix,
+    * @param measurementNoiseCovDiag - value of elements on main diagonal measurement noise cross-covariance matrix,
+    * @param _f - function for computing the next state from the previous state,
+    * @param _h - function for computing the measurement from the state,
+    * @param type - type of the created matrices that should be CV_32F or CV_64F.
+    */
+    CV_WRAP UnscentedKalmanFilterParams( int dp, int mp, int cp, double processNoiseCovDiag, double measurementNoiseCovDiag,
+                                UKFStateFunction _f, UKFMeasFunction _h, int type = CV_64F );
+
+    /** The function for initialization of Unscented Kalman filter
+    * @param dp - dimensionality of the state vector,
+    * @param mp - dimensionality of the measurement vector,
+    * @param cp - dimensionality of the control vector,
+    * @param processNoiseCovDiag - value of elements on main diagonal process noise cross-covariance matrix,
+    * @param measurementNoiseCovDiag - value of elements on main diagonal measurement noise cross-covariance matrix,
+    * @param _f - function for computing the next state from the previous state,
+    * @param _h - function for computing the measurement from the state,
+    * @param type - type of the created matrices that should be CV_32F or CV_64F.
+    */
+    CV_WRAP void init( int dp, int mp, int cp, double processNoiseCovDiag, double measurementNoiseCovDiag,
```

Comment:
same thing. In fact, please, remove CV_PROP_... and replace CV_EXPORTS_W with CV_EXPORTS, because this class is only usable from C/C++